ekm/signer_key_manager_test.go

```diff
@@ -279,6 +293,284 @@ func TestSlashing(t *testing.T) {
 	})
 }
 
+func TestSignBeaconObject(t *testing.T) {
+	km := testKeyManager(t, nil)
+
+	sk1 := &bls.SecretKey{}
+	require.NoError(t, sk1.SetHexString(sk1Str))
+	require.NoError(t, km.AddShare(sk1))
+
+	currentSlot := km.(*ethKeyManagerSigner).storage.Network().EstimatedCurrentSlot()
+	highestProposal := currentSlot + minSPProposalSlotGap + 1
+
+	t.Run("Sign Deneb block", func(t *testing.T) {
+		var beaconBlock = &deneb.BeaconBlock{
+			Slot:          highestProposal,
+			ProposerIndex: 0,
+			ParentRoot: phase0.Root{
```

Comment:
all these byte arrays look bulky, what do you think about extracting them to a function/method to keep test body relatively short?